composer.json

```diff
@@ -31,7 +31,7 @@
 	"require": {
 		"php": ">=5.3.0",
 		"data-values/data-values": "~1.0|~0.1",
-		"data-values/interfaces": "~0.1",
+		"data-values/interfaces": "~0.1.5",
```

Comment:
> Not sure about the syntax, but since the test succeed it obviously does what I wanted it to do.

Logical fallacy. And in this case easily shown wrong. Remove the change and it will still work.

If all you want to do is increase the lower bound, then `0.*,>=0.1.5` does the job. What you did also decreases the upper bound.